script.js

```diff
@@ -0,0 +1,86 @@
+const playButton = document.querySelector('.playButton')
+function shuffleArray(array) {
+    for (var i = array.length - 1; i > 0; i--) {
+        var j = Math.floor(Math.random() * (i + 1));
+        var temp = array[i];
+        array[i] = array[j];
+        array[j] = temp;
+    }
+}
+fetch('./films.json')
+    .then((response) => response.json())
+    .then((json) => {
+
+        const playGame = () => {
```

Comment:
I meant to create the function definition outside of the `fetch` block, and just call it inside here instead